imagely.es6

```diff
@@ -0,0 +1,242 @@
+import _ from 'lodash';
+import Promise from 'bluebird';
+import imageSize from 'image-size';
+import path from 'path';
+import phantom from 'phantom';
+import phantomjs from 'phantomjs';
+import request from 'request-promise';
+const fs = Promise.promisifyAll(require('fs'));
+
+/**
+ * Renders a source HTML file as an image.
+ *
+ * @public
```

Comment:
I mean, you are passing it to the class:
`new Imagely(source, destination, options, callback);`

Anyways, moved.